### PR TITLE
Correct structure check for unowned planet mines

### LINKED ISSUE #1283: Mines can't destroy ships
Version: 387a65d513bf379ae

Save: http://dump.bitcheese.net/files/gukaseh/undying.mps

Voker57 empire. At the end of turn, ship at Rheims 3 will die and resurrect.

### PR DESCRIPTION
**Pull request by dbenage-cx**
Fixes #1283

### CODE REVIEW

File: default/scripting/techs/defense/mines.macros [hunk 1]
```diff
@@ -80,7 +80,7 @@ EG_SYSTEM_MINES
                     Not HasTag name = "UNOWNED_FRIENDLY"
                     ]
                 ]
-                Structure low = @1@ + 0.0001
+                Structure high = @1@ + 0.0001
```
Comment (Dilvish-fo) [inline default/scripting/techs/defense/mines.macros]: seems to me that this line should be the same as line 63 in the empire-owned planet case,
``` Structure high = @1@```

Comment (dbenage-cx) [reply default/scripting/techs/defense/mines.macros]: TY, fixed

**Comment by Dilvish-fo** [comment]
The underlying problem is rarely enough encountered that unless someone had a saved game to test this against (other than the one posted with the cited issue that you already have access to), or was willing to put time into engineering up a test case, that any additional testing of this is not likely to occur.  The fix is clear and straightforward enough that I would suggest it is ready to merge.

**Comment by Vezzra** [comment]
@dbenage-cx, I can't open the provided savegame, neither on OSX nor on Linux, so I can't test it, but assuming you did test it on your system I agree with @Dilvish-fo. As far as I'm concerned, go ahead and merge.

**Comment by dbenage-cx** [comment]
Ok, since it is last in the order skipping extensive testing for edge cases.
